lambda/custom/index.js

```diff
@@ -15,29 +15,30 @@ const APL_DOC = require ('./document/renderPage.json' ) ;
 const TWO_PAGER_COMMANDS =  require ('./document/twoSpeakItemsCommand.json' ) ;
 const ONE_PAGER_COMMANDS =  require ('./document/oneSpeakItemCommand.json' ) ;
 const TOKEN_ID = 'pagerSample';
-let firstTransformer = [
+let firstTransformerList = [
```

Comment:
If this is constant, can you make it so? Same for the below list.

Reply:
done